python/API_Examples_Python.py

```diff
@@ -0,0 +1,200 @@
+import json     # python3 -m pip install simplejson
```

Comment:
Add a shebang line `#!/usr/bin/env python3`
See https://stackoverflow.com/questions/7670303/purpose-of-usr-bin-python3

Reply:
done. Thanks for the link!